src/controllers/author.js

```diff
@@ -0,0 +1,27 @@
+// src/controllers/author.js
+const {
+  getAllItems,
+  createItem,
+  updateItem,
+  getItemById,
+  deleteItem,
+} = require('./helpers');
+
+const getAuthors = (_, res) => getAllItems(res, 'author');
+
+const createAuthor = (req, res) => createItem(res, 'author', req.body);
```

Comment:
you could destructure off the req here like ({ body }, res) => etc

The way you have done it is fine though